build.rs

```diff
@@ -52,6 +52,11 @@ type Date = &'static str;
 /// versions for which to enable particular features.
 type ConditionalCfg = (Cfg, &'static [(Date, Cfg)]);
 const CONDITIONAL_CFGS: &'static [ConditionalCfg] = &[
+	(None, &[("2019-02-24", Some("pattern_guards"))]),
+	(None, &[("2018-08-14", Some("non_exhaustive"))]),
+	(Some("collections"), &[("2018-08-13", None)]),
+	(Some("str_char"), &[("2018-08-13", None)]),
+	(Some("unicode"), &[("2018-08-13", None)]),
```

Comment:
Combine this into a single cfg.

Reply:
`str_char` and `unicode` features are now enabled by the `unicode` cfg.